### ISSUE TITLE
Create [conformance] tags for extended.test

### ISSUE THREAD

**Issue by jayunit100** [openshift/origin#7952]
**Requirement** 

Validating a base level of functionality for an openshift cluster, in under 20 minutes.

**Problem**  

Validating an openshift cluster, even if skipping Upstream tests, can take 100s of minutes, and possibly hang.

**Example**

The following test run from today hung after well over an hour (even after filtering out all upstream tests using synthetic skips).

```
Ran 216 of 381 Specs in 6213.299 seconds
FAIL! -- 40 Passed | 50 Failed | 0 Pending | 165 Skipped [ec2-user@ip-172-18-2-154 ~]$
```

**Solution** 

Minimize the amount of tests we need to run to characterize a base working cluster by annotating a basis set that can be selected via ginkgo.focus.

**Implementation**
- Label downstream conformance tests with a ginkgo skippable/focusable `[conformance]` tag.
- some tests (i.e. `openshift can execute jobs` or `openshift routers` , and maybe `openshift metrics` (once it exists), should have a `[conformance]` label added on.

_this is for large clusters, where our goal is to minimize the amount of testing time without sacrificing coverage_.

So, to complete this ticket.
- identify a "critical" subset tests inside of extended.test
- tag them as `[conformance]`

**Comment by jayunit100** [comment]
cc @timothysc

**Comment by timothysc** [comment]
> this is for large clusters, where our goal is to minimize the amount of testing time without sacrificing coverage.

Can't not state enough how much this is needed... Validating a large installation takes days atm b/c running the entire suite takes days.  

/cc @jeremyeder

**Comment by stevekuznetsov** [comment]
Being addressed in #13551

**Comment by stevekuznetsov** [comment]
Addressed by the linked PR

/close
